pages/enterprise/setup.rst

```diff
@@ -42,67 +42,85 @@ The following list shows the minimum required Graylog versions for the Graylog E
 Installation
 ============
 
-Once you `purchase a license for Graylog Enterprise <https://www.graylog.org/enterprise/>`_,
-you will get download links for the tarballs and DEB/RPM packages in the confirmation mail.
+Since Graylog 2.4 the Graylog Enterprise Plugins can be installed the same way Graylog is installed. In most setups this will be done with the package tool provided by the distribution you are using and the online repository.
 
-.. note:: The Graylog Enterprise plugins need to be installed on all your Graylog nodes!
+.. note:: For previous versions of Graylog Enterprise please contact your support representive.
 
-Tarball
--------
+Once you installed the Graylog Enterprise Plugins you need to optain a License from `the Graylog Enterprise Page <https://www.graylog.org/enterprise/>`_. 
 
-The tarball includes the enterprise plugin JAR files.
+Should a simple `apt-get install graylog-enterprise-plugins` or `yum install graylog-enterprise-plugins` not work for you, the following information might help you.
```

Comment:
I think we should document the `apt-get` and `yum` install methods a bit more than with one sentence. Maybe just replace the `dpkg -i ` and `rpm -Uvh` sections below with the `apt-get` and `yum` instructions.

Installing the packages via the repository should be the main way of doing it.